code/src/components/App.js

```diff
@@ -0,0 +1,71 @@
+/* eslint max-len: ["error", { "code": 180 }] */
+
+import React, { useState } from 'react';
+
+import UserName from 'components/UserName';
+import CoolestThing from 'components/CoolestThing';
+import PerfectJob, { defaultPerfectJobOption } from 'components/PerfectJob';
+import Fondness from 'components/Fondness';
+import Summary from 'components/Summary';
+
+const App = () => {
+  const stepsTotalCount = 4
+  const [step, setStep] = useState(1);
+  const [userName, setUserName] = useState('');
+  const [coolestThing, setCoolestThing] = useState('How cute and fluffy they are');
+  const [perfectJob, setPerfectJob] = useState(defaultPerfectJobOption);
+  const [fondness, setFondness] = useState(5);
+
+  const handleStepIncrease = () => {
+    if (step === 1 && userName === '') {
+      return
+    }
+    setStep(step + 1);
+  }
+
+  return (
```

Comment:
Your code is very clean and easy to follow.

Reply:
Thank you 😊